client/renderers/EGL/egl.c

```diff
@@ -145,6 +138,18 @@ static struct Option egl_options[] =
   {0}
 };
 
+static struct Option opengl_options_x11[] =
+{
+  {
+    .module       = "egl",
+    .name         = "vsync",
+    .description  = "Enable vsync",
+    .type         = OPTION_TYPE_BOOL,
+    .value.x_bool = false
+  },
+  {0}
+};
+
```

Comment:
No, keep this part of the standard EGL options. vsync disable is not just `x11` or `Wayland`. It would be better to add a validator that warns/errors if it's enabled on Wayland if it really warrants this.